.prettierignore

```diff
@@ -0,0 +1,2 @@
+tests/
+testing/
```

Comment:
I don't have these directories, are they the output of running the test profile? And should they really be part of this config file then?

Reply:
This is for prettier, and is based on the nf-core standard .gitignore